src/Components/Img.jsx

```diff
@@ -4,7 +4,7 @@ const Img = props => (
   <li className="gif-wrap">
     {console.log("Img's array data: ", props)}
     <h1>{props.url}</h1>
-    {/* <img src={props} alt="" /> */}
+    <img src={props.url} alt="" />
```

Comment:
All the headers were showing me the URLs, so we're using that as the image's src